apps/passport-client/components/screens/ChangePasswordScreen.tsx

```diff
@@ -18,6 +19,8 @@ import { PasswordInput } from "../shared/PasswordInput";
 
 export function ChangePasswordScreen() {
   const self = useSelf();
+  const hasSetupPassword = useHasSetupPassword();
+  const [isChangePassword] = useState(hasSetupPassword); // This value should only change on load, not on render
```

Comment:
i don't understand this comment

Reply:
after we successfully add a password (adding a salt and removing the encryption_key), `hasSetupPassword` will return true on the next render. however, this would mean that our "finished" state screen would think that we just changed our password, rather than adding a new password. we don't want `isChangePassword` to change every render